tests/cover/test_cathetus.py

```diff
@@ -0,0 +1,193 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2018 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import sys
+import math
+
+from hypothesis.internal.cathetus import cathetus
+
+
+def assert_cathetus_exact(h, a, b):
```

Comment:
Do we really need to provide our own test helpers here? I thought pytest did introspection like this for us.

If I replace this helper with `assert cathetus(h, a) == b` and break the implementation, this is the sort of output I get from pytest:

```traceback
Traceback (most recent call last):
  File "/Users/alexwlchan/repos/hypothesis-python/tests/cover/test_cathetus.py", line 193, in test_pythagorean_triples
    assert_cathetus_exact(h, a, b)
  File "/Users/alexwlchan/repos/hypothesis-python/tests/cover/test_cathetus.py", line 27, in assert_cathetus_exact
    assert cathetus(h, a) == b
AssertionError: assert 8.0 == 4
 +  where 8.0 = cathetus(5, 3)
```